modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/type/ApplicationModelTypeUtils.java

```diff
@@ -236,38 +238,65 @@ private static void handleNestedParameters(ComponentModel componentModel, Stream
                                              Multimap<ComponentIdentifier, ComponentModel> nestedComponents,
                                              ExtensionModelHelper extensionModelHelper,
                                              ParameterizedModel model, Predicate<ParameterModel> parameterModelFilter) {
-    childrenComponentModels
-        .forEach(childComp -> extensionModelHelper.findParameterModel(childComp.getIdentifier(), model)
-            .filter(parameterModelFilter::test)
-            // do not handle the callback parameters from the sources
-            .filter(paramModel -> {
-              if (model instanceof SourceModel) {
-                return !(((SourceModel) model).getSuccessCallback()
-                    .map(sc -> sc.getAllParameterModels().contains(paramModel))
-                    .orElse(false) ||
-                    ((SourceModel) model).getErrorCallback()
-                        .map(ec -> ec.getAllParameterModels().contains(paramModel))
-                        .orElse(false));
-              } else {
-                return true;
-              }
-            })
-            .filter(paramModel -> paramModel.getDslConfiguration().allowsInlineDefinition())
-            .ifPresent(paramModel -> {
-              if (isContent(paramModel)) {
-                componentModel.setParameter(paramModel,
-                                            new DefaultComponentParameterAst(trim(((ComponentModel) childComp)
-                                                .getTextContent()),
-                                                                             () -> paramModel, childComp.getMetadata()));
-              } else {
-                enrichComponentModels(componentModel, nestedComponents,
-                                      of(extensionModelHelper.resolveDslElementModel(paramModel,
-                                                                                     componentModel
-                                                                                         .getIdentifier())),
-                                      paramModel, extensionModelHelper);
-              }
-
-            }));
+    model.getAllParameterModels()
```

Comment:
seems like nothing change here other than formatting? Is taht the case? Can it be reverted then?

Reply:
no, i turned this around, before it iterated the children and matched those against the parameter models, now it iterates the parameter models and looks it up in the dsl.